### PR TITLE
Fix positioning of the public page footer

### LINKED ISSUE #34399: [Bug]: Public view footer broken
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

On latest master (26), public view footer is broken du to #34331.

1. the removed `position: sticky` property from `<footer>`, results in the footer being displayed on the top of the page.
2. The `- 65px` is not enough because `<footer>` actually takes 65px height plus 2x 8px padding on top and bottom. So this should probably be `65px + 8px + 8px` = `81px`.

@szaimen because you tried to fix public view footer in #34331.

### Steps to reproduce

1. Open a document in public share
2. Watch the footer being broken
3. Add `position: sticky` back to `footer` CSS
4. Watch the footer overlapping long content

### Expected behavior

Footer should be at the bottom and not overlap long content.

### Installation method

_No response_

### Operating system

_No response_

### PHP engine version

_No response_

### Web server

_No response_

### Database engine version

_No response_

### Is this bug present after an update or on a fresh install?

_No response_

### Are you using the Nextcloud Server Encryption module?

_No response_

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

_No response_

### List of activated Apps

```shell
Unrelated
```

### Nextcloud Signing status

_No response_

### Nextcloud Logs

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by juliusknorr**
Fix: #34399 

Fixes the regression mentioned in https://github.com/nextcloud/server/pull/34331 that the footer would be positioned a the page top.

### CODE REVIEW

File: core/css/public.scss [hunk 1]
```diff
@@ -69,7 +69,7 @@ $footer-height: 65px;
 
 	/* public footer */
 	footer {
-		position: sticky !important;
+		position: fixed;
```
Comment (szaimen) [inline core/css/public.scss:72]: Can we put sticky then into the files_sharing public.scss? Would be cool if it would be sticky at the bottom!

**Review by szaimen — CHANGES_REQUESTED** [review]

File: core/css/public.scss (hunk 1 above)
Comment (juliushaertl) [reply core/css/public.scss:72]: I'd vote to revert it for now and for properly making it sticky, move the footer container into the scroll area instead.

**Review by vinicius73 — APPROVED** [review]
Revert it for now to fix the broken CI

File: apps/files_sharing/css/public.scss [hunk 2]
```diff
@@ -82,6 +83,11 @@ $download-button-section-height: 200px;
 	max-height: 100%;
 }
 
+
+.app-files_sharing #app-content footer {
+	position: sticky !important;
```
Comment (szaimen) [inline apps/files_sharing/css/public.scss:88]: do you think this will work or will we need to make an exceptioni here for txt or md files?

Comment (szaimen) [reply apps/files_sharing/css/public.scss:88]: at least it works for other file types in my testing...

**Comment by vinicius73** [comment]
Works @szaimen <3

**Comment by vinicius73** [comment]
image

It this CI error related to this PR @juliushaertl @szaimen

**Review by szaimen — APPROVED** [review]

**Pull request merged**